scanners/reactnative/plain.go

```diff
@@ -210,47 +210,47 @@ func (scanner *Scanner) configs(isPrivateRepo bool) (models.BitriseConfigMap, er
 	}
 
 	// ios cd
-	if scanner.iosScanner != nil {
-		for _, descriptor := range scanner.iosScanner.ConfigDescriptors {
```

Comment:
Removed this loop over iOS descriptors to prevent duplicate Steps added to the config. Use the first descriptor only.